File: MedicPortal/Data/Models/Patient.cs
using System;
using System.Collections.Generic;

namespace MedicPortal.Data.Models
{
    public class Patient
    {
        public string Id { get; set; } = Guid.NewGuid().ToString();
        public string AppUserId { get; set; }
        public AppUser AppUser { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public DateTime Birthdate { get; set; }
        public string Address { get; set; }
        public string PhoneNumber { get; set; }
        public bool IsDeleted { get; set; }
        public DateTime? Deleted { get; set; }
        public AppUser DeletedBy { get; set; }
        public string DeletedById { get;set; }
        public string AvatarImgSrc { get; set; }
    }
}

File: MedicPortal/PortalApp/src/app/core/entities/patient.ts
import { parse } from 'date-fns';

export class Patient {
    id: string;
    firstName: string;
    lastName: string;
    birthdate: Date;
    address: string;
    phoneNumber: string;
    avatarImgSrc: string;
    constructor(serverObject?: any) {
        if (serverObject) {
            this.id = serverObject.id;
            this.firstName = serverObject.firstName;
            this.lastName = serverObject.lastName;
            this.birthdate = serverObject.birthdate;
            this.address = serverObject.address;
            this.phoneNumber = serverObject.phoneNumber;
            this.avatarImgSrc = serverObject.avatarImgSrc;
        }
    }
}
